server/webhook_worker.go

```diff
@@ -48,8 +50,46 @@ func (ww webhookWorker) process(msg *webhookMessage) (err error) {
 	if err = v.JiraWebhook.expandIssue(ww.p, msg.InstanceID); err != nil {
 		return err
 	}
+	// We will only send webhook events if we have a connected instance.
+	instance, err := ww.p.instanceStore.LoadInstance(msg.InstanceID)
+	if err != nil {
+		// This isn't an internal server error. There's just no instance installed.
+		return err
+	}
+
+	comments := v.JiraWebhook.Issue.Fields.Comments.Comments
+	mattermostUserID, err := ww.p.userStore.LoadMattermostUserID(instance.GetID(), comments[len(comments)-1].Author.AccountID)
+	if err != nil {
+		return err
+	}
+
+	// Check if the user has permissions.
+	c, err := ww.p.userStore.LoadConnection(instance.GetID(), mattermostUserID)
+	if err != nil {
+		// Not connected to Jira, so can't check permissions
+		return err
+	}
+
+	client, err := instance.GetClient(c)
+	if err != nil {
+		return err
+	}
+
+	// If this is a comment-related webhook, we need to check if they have permissions to read that.
```

Comment:
```suggestion
	// If this is a comment-related webhook payload, we need to check if they have permission to read that.
```
read what and who is they here don't use this, that and they in comments specify the things you are talking about comments should be detailed